winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+                    <TextBlock Text="{Binding Title}" TextAlignment="Center" FontWeight="Bold" FontSize="14" />
+                    <TextBlock Text="{Binding Description}" TextAlignment="Center" FontSize="14" TextWrapping="Wrap" />
+                </StackPanel>
+            </Border>
+        </DataTemplate>
+    </kanban:SfKanban.CardTemplate>
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+### Customize kanban cards using template selector
+
+To dynamically apply different templates to Kanban cards based on specific conditions, use the `CardTemplateSelector` property. This allows you to define multiple templates and choose one at runtime based on card data.
+
+{% tabs %}
+{% highlight XAML hl_lines="28" %}
+
+<Grid>
+    <Grid.DataContext>
+        <local:ViewModel/>
+    </Grid.DataContext>    
+    <Grid.Resources>
+        <DataTemplate x:Key="highPriorityTemplate">
+            <Border BorderBrush="Red" BorderThickness="2" CornerRadius="10" Background="LightPink">
+                <StackPanel Margin="10">
+                    <TextBlock Text="{Binding Title}" FontWeight="Bold" FontSize="16" />
+                    <TextBlock Text="{Binding Description}" FontSize="14" TextWrapping="Wrap" />
+                </StackPanel>
+            </Border>
+        </DataTemplate>
+        <DataTemplate x:Key="defaultTemplate">
+            <Border BorderBrush="Green" BorderThickness="1" CornerRadius="10" Background="LightGreen">
+                <StackPanel Margin="10">
+                    <TextBlock Text="{Binding Title}" FontWeight="Bold" FontSize="16" />
+                    <TextBlock Text="{Binding Description}" FontSize="14" TextWrapping="Wrap" />
+                </StackPanel>
+            </Border>
+        </DataTemplate>
+        <local:KanbanCardTemplateSelector x:Key="cardTemplateSelector"
+                                          HighPriorityTemplate="{StaticResource highPriorityTemplate}"
+                                          DefaultTemplate="{StaticResource defaultTemplate}"/>
+    </Grid.Resources>
+
+    <kanban:SfKanban x:Name="kanban" 
+                     CardTemplateSelector="{StaticResource cardTemplateSelector}"
+                     ItemsSource="{Binding TaskDetails}">
+    </kanban:SfKanban>
+</Grid>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="KanbanCardTemplateSelector.cs" %}
+
+public class KanbanCardTemplateSelector : DataTemplateSelector
+{
+    public DataTemplate HighPriorityTemplate { get; set; }
+    public DataTemplate DefaultTemplate { get; set; }
+
+    protected override DataTemplate SelectTemplateCore(object item, DependencyObject container)
+    {
+        var task = item as KanbanModel;
+        if (task != null && task.IndicatorColorKey == "High")
+        {
+            return HighPriorityTemplate;
+        }
+
+        return DefaultTemplate;
+    }
+}
+
+{% endhighlight %}
```

Comment:
Add notes:

The `DataContext` for both the `CardTemplate` and `CardTemplateSelector` properties in the `SfKanban` is set to `KanbanModel`.

Reply:
Added.